tests/expected/all.json

```diff
@@ -653,8 +775,8 @@
         "test/Reset.t.sol:ResetTest": [
             {
                 "name": "checkFoo()",
-                "exitcode": 1,
```

Comment:
ah so the test used to fail, but now with the custom halmos annotation it passes? Amazing 🥳

Reply:
yes, exactly.